src/server/views/Html.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+
+interface JavaScriptProps {
+    sources: string[] | undefined;
+}
+
+interface StyleSheetProps {
+    hrefs: string[] | undefined;
+}
+
+interface HtmlProps {
+    children: React.ReactNode;
+    files: Express.Files | undefined;
+}
+
+const JavaScript = (props: JavaScriptProps) => {
+    const files = props.sources
+        ? props.sources.map((src, index) => <script key={index} type="text/javascript" src={src} />)
+        : undefined;
+    return (
+        <React.Fragment>
+            {files}
+        </React.Fragment>
+    );
+};
+
+const StyleSheet = (props: StyleSheetProps) => {
+    const files = props.hrefs
+        ? props.hrefs.map((href, index) => <link key={index} rel="stylesheet" type="text/css" href={href} />)
+        : undefined;
+    return (
+        <React.Fragment>
+            {files}
+        </React.Fragment>
+    );
+};
+
+const Html = (props: HtmlProps) => {
+    return (
+        <html lang="en">
```

Comment:
May want to add a TODO to handle `lang` switching in the future.

Reply:
Will do